charts/radar-upload-source-connector/values.yaml

```diff
@@ -67,6 +67,9 @@ affinity: {}
 zookeeper: cp-zookeeper-headless:2181
 # -- Kafka broker URLs
 kafka: PLAINTEXT://cp-kafka-headless:9092
+environment:
```

Comment:
Perhaps add a comment above to document how you can use the `environment` variable.